examples/load-data/README.md

```diff
@@ -0,0 +1,17 @@
+# Connect Saturn Cloud to Your Data
+
+Saturn does not host data, but we know that accessing your data is vital for your data science practice. We have several options for linking your data sources to your Saturn Cloud account. If your data source is not listed here, please contact us at hello@saturncloud.io or on our community Slack and we’ll help you!
```

Comment:
1. We should always say `Saturn Cloud` and never just `Saturn`
2. If people have trouble we should direct them to intercom
3. We will "sort of" host data with the shared folders, so I am gonna change the wording so we don't forget to adjust later

```suggestion
Saturn Cloud resources need to pull data from external locations and we know that accessing your data is vital for your data science practice. We have several options for loading your data sources into your Saturn Cloud resources. If your data source is not listed here and you need help connecting, please reach out using the chat option in the lower right-hand corner of any Saturn Cloud page.
```